packages/survey-creator-core/src/components/tabs/designer.ts

```diff
@@ -94,10 +94,12 @@ export class TabDesignerViewModel extends Base {
 
     this.actionContainer.setItems([{
       id: "collapseAll",
+      tooltip: getLocString("ed.collapseAllTooltip"),
```

Comment:
In order to be updated on locale changed you need ot use the `locTooltipName` property here